src/modules/commander/Commander.cpp

```diff
@@ -658,10 +712,10 @@ Commander::handle_command(const vehicle_command_s &cmd)
 
 				bool cmd_arms = (static_cast<int>(cmd.param1 + 0.5f) == 1);
```

Comment:
```suggestion
				const bool cmd_arms = (static_cast<int>(cmd.param1 + 0.5f) == 1);
```
Just saw this. Casting the parameters could also be done just once at the beginning.

Reply:
Also why are these + 0.5 while `roundf` is used for the other integer command parameters?

Reply:
No real reason, typical legacy cruft.

Reply:
Fixed in f6a00eb